BaseTools/ImageTool/ElfScan.c

```diff
@@ -540,11 +533,16 @@ CreateIntermediate (
         return RETURN_VOLUME_CORRUPTED;
       }
 
+    if (!ProcessRelocSection (Shdr)) {
+      continue;
+    }
```

Comment:
These 3 strings need 2 more spaces for proper alignment.

Reply:
Fixed, tyvm